build.sbt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ThisBuild / name         := "fa-db"
+ThisBuild / organization := "za.co.absa"
+
+lazy val scala211 = "2.11.12"
+lazy val scala212 = "2.12.12"
```

Comment:
This library could in theory support 2.13 as well, right?

Reply:
Depends on the lib(s) used to actually access the DB (Slick, Doobie,...) if they support it.
Otherwise I don't see reason why not. 😉